docs/umbraco/unit-testing.md

```diff
@@ -152,6 +152,29 @@ dictionary
     .AddDictionaryValue("myKey", "myValue", 1); // Setting languageId to 1 - English US
 ```
 
+## Testing content is overridden  
+Blocks in Umbraco can be overridden with blocks generated by your code. This is typically done when populating a summary list with data the user has entered or replacing a token in a string with a value. You should write tests to prove this happens correctly. 
```

Comment:
While this first sentence is true, the more common scenario and the one you're describing is that "Properties in Umbraco can be overridden with values generated by your code."

Also, I misread the heading as saying that "testing content" had been overridden. "Test that content is overridden" would be clearer.